Detectors/TRD/reconstruction/src/TrackletsParser.cxx

```diff
@@ -145,7 +145,7 @@ int TrackletsParser::Parse()
       // digit half chamber header ends with 01b and has the supermodule in position (9-13).
       // this of course can conflict with a tracklet hc header, hence should not be used!
       // NBNBNBNB
-      if ((tmpheader & 0x3) == 0x1 && (((tmpheader >> 9) & 0x1f) == mHCID / 30)) {
+      if ((tmpheader & 0x3) == 0x1 && (((tmpheader >> 9) & 0x1f) == mHCID / constants::NHCPERSEC)) {
```

Comment:
Could we not get rid of tmpheader and use the hcheader fields.
This would get rid of the bitshifting and a useless variable?

Reply:
There are quite a few things I would like to change in the tracklet and digit parsers as well. But as I said above I wanted to leave them as they are for the moment so that this PR does not get too big. Sean also has some changes in the pipeline and after that I will look in more detail also through the parsing part